int16 carID
---
int32 distanceSoFar
int16 carID
